<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebediaClass</class>
 <widget class="QMainWindow" name="WebediaClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Webedia</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_afficheresultat">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_bdd">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>214</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_equipement">
     <item>
      <widget class="QLabel" name="label_ajout_equipement">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>             Ajout d'un equipement</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_nom_equipement_2">
         <property name="text">
          <string>Nom equipement :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_adresse_equipement">
         <property name="text">
          <string>Adresse :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_nom_equipement"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_adresse_equipement"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ajout_equipement">
       <property name="text">
        <string>Ajout d'un equipement</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>201</width>
      <height>155</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label_equipement">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Equipement :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_affiche_creation_equipement">
       <property name="text">
        <string>Creation</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_affiche_supprimer_equipement">
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_affiche_equipement">
       <property name="text">
        <string>Affiche</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_module">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Module :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_affiche_modifier_module">
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_affiche_module">
       <property name="text">
        <string>Affiche</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_affiche_creation_module">
       <property name="text">
        <string>Creation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_affiche_modifier_equipement">
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_affiche_supprimer_module">
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCr_ation">
   <property name="text">
    <string>Ajout</string>
   </property>
  </action>
  <action name="actionModifier">
   <property name="text">
    <string>Modifier</string>
   </property>
  </action>
  <action name="actionSupprimer">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionCr_ation_2">
   <property name="text">
    <string>Cr√©ation</string>
   </property>
  </action>
  <action name="actionModifier_2">
   <property name="text">
    <string>Modifier</string>
   </property>
  </action>
  <action name="actionSupprimer_2">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionAffiche">
   <property name="text">
    <string>Affiche</string>
   </property>
  </action>
  <action name="actionAffiche_2">
   <property name="text">
    <string>Affiche</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Webedia.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_ajout_equipement</sender>
   <signal>clicked()</signal>
   <receiver>WebediaClass</receiver>
   <slot>onAjoutEquipementButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>785</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCreationButtonClicked()</slot>
  <slot>onListWidgetClicked()</slot>
  <slot>onAjoutEquipementButtonClicked()</slot>
  <slot>creationModule()</slot>
 </slots>
</ui>
